package.json

```diff
@@ -7,22 +7,44 @@
     "type": "git",
     "url": "git://github.com/ssbc/ssb-buttwoo.git"
   },
+  "files": [
+    "*.js",
+    "package.json.license",
+    "LICENSES/*",
+    "!example.js"
+  ],
   "dependencies": {
     "bipf": "^1.7.0",
     "blake3": "^2.1.7",
     "fast-varint": "^1.0.1",
     "monotonic-timestamp": "^0.0.9",
     "ssb-bfe": "^3.3.0",
-    "ssb-keys": "^8.3.0"
+    "ssb-keys": "^8.3.0",
```

Comment:
can we bump this to 8.4? tests are not running with 8.3

Reply:
Done